packages/types-algosdk/index.d.ts

```diff
@@ -21,6 +21,42 @@ declare module 'algosdk' {
     accountInformation(address: string): Action<AccountInfo>
   }
 
+  export const OnApplicationComplete: {
+    ClearStateOC: number;
+    CloseOutOC: number;
+    DeleteApplicationOC: number;
+    NoOpOC: number;
+    OptInOC: number;
+    UpdateApplicationOC: number;
+	};
```

Comment:
indentation

Reply:
interesting, i am seeing indented code in my visual code.
let me recommit again this file

Reply:
shows correctly with `viewfile` now.